mysql/datadog_checks/mysql/statement_samples.py

```diff
@@ -461,7 +474,6 @@ def _collect_plan_for_statement(self, row):
                 'mysql': {k: v for k, v in row.items() if k not in EVENTS_STATEMENTS_SAMPLE_EXCLUDE_KEYS},
             }
 
-    @tracked_method(agent_check_getter=agent_check_getter)
```

Comment:
removing this because it doesn't seem to work - no metrics show up for this. Possibly because the function yields instead of returning? but possibly no tracked_method metrics are showing up from the mysql checks?